dev/archery/archery/integration/datagen.py

```diff
@@ -193,6 +193,28 @@ def generate_range(self, size, lower, upper, name=None,
         return PrimitiveColumn(name, size, is_valid, values)
 
 
+# Integer field that fulfils the requirements for the run ends field of RLE.
+# The integers are positive and in a strictly increasing sequence
+class RunEndsField(IntegerField):
+    def __init__(self, name, bit_width, *, nullable=False,
+                 metadata=None):
+        super().__init__(name, is_signed=True, bit_width=bit_width,
+                         nullable=nullable, metadata=metadata, min_value=1)
```

Comment:
As i mentioned in my comment about starting on a non-zero value, it's technically meaningless and incorrect to start on a 0 value for run-ends, the min-value should be 1 since run-ends are always 1 past the last index. Run-ends should never start on a 0.

Reply:
sure i'll add the comment to the RunEndsField